pkgs/development/compilers/elm/default.nix

```
@@ -7,7 +7,7 @@ let
   fetchElmDeps = import ./fetchElmDeps.nix { inherit stdenv lib fetchurl; };
 
   hsPkgs = haskell.packages.ghc883.override {
-    overrides = self: super: with haskell.lib;
+    overrides = self: super: with haskell.lib; with stdenv.lib;
```

```suggestion
    overrides = self: super: with haskell.lib; with lib;
```

by recent contribution change.